core/factory.py

```diff
@@ -376,7 +396,7 @@ def is_commander(self, cid, uid) -> bool:
 
 # command handlers
 @bot.message_handler(commands=[constants.COMMAND_START, constants.COMMAND_CANCEL, constants.COMMAND_TEST])
-@lock_method_for_strangers(checker=bot.is_member, default=bot.notify_about_membership)
+@lock_method_for_strangers(checker=bot.is_member, stop_bloggers=bot.is_blogger, default=bot.notify_about_membership)
```

Comment:
We shouldn't be modifying the existing decorator. Instead, we should re-apply it with a separate checker. We want something like this:
```python
@lock_method_for_strangers(checker=bot.is_member, default=bot.notify_about_membership)
@lock_method_for_strangers(checker=bot.is_blogger, default=bot.notify_about_violation)
def command_handler(message):
```